### ISSUE TITLE
Error while indenting .inst.in files

### ISSUE THREAD

**Issue by bangerth** [dealii/dealii#2985]
Since #2928 , we now also indent the .inst.in files. But on this newfangled system of mine, I get this error:
```
[1/1] Indenting all files in the deal.II directories
FAILED: CMakeFiles/indent
cd /home/fac/f/bangerth/p/deal.II/1/dealii && ./contrib/utilities/indent
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
_: format_inst: command not found
```

@tjhei -- since you wrote this, any idea? What confuses me slightly is that you execute this thing:
```
format_inst()
{
    f=$1
    cp $f $f.tmp
    sed -i.orig 's#\\{#{ //#g' $f.tmp
    sed -i.orig 's#\\}#} //#g' $f.tmp
    astyle --quiet $f.tmp
    sed -i.orig 's#{ //#\\{#g' $f.tmp
    sed -i.orig 's#} //#\\}#g' $f.tmp
    if ! diff -q $f $f.tmp >/dev/null
    then
        cp $f.tmp $f
    fi
    rm $f.tmp $f.tmp.orig
}

export -f format_inst
find source -name '*.inst.in' | xargs -n 1 -P 10 -I % bash -c 'format_inst "$@"' _ %
```
Apparently the child `bash` does not inherit the `format_inst` function.

**Comment by tjhei** [comment]
I have no clue. Do you know of a different way to execute several commands in a row that can be executed by xargs in parallel?

**Comment by bangerth** [comment]
I've pushed a third commit that documents what's happening inside `operator<< / >>` a bit better.

**Comment by bangerth** [comment]
Workaround in #2988.

**Comment by bangerth** [comment]
I opted for the simple solution of just not doing it in parallel. We have few of these files, and they are small -- together just 11k lines.

**Comment by bangerth** [comment]
Fixed by #2988.
